.travis.yml

```diff
@@ -1,9 +1,22 @@
 sudo: required
+cache: bundler
+language: ruby
+
 services:
   - docker
 
 install:
-  - export BUNDLE_GEMFILE=$PWD/Gemfile
   - bundle install
 
-script: bundle exec kitchen verify ${INSTANCE}
+env:
+  matrix:
+    - INSTANCE: centos-7
+    - INSTANCE: debian-9
+    - INSTANCE: ubuntu-1804
+
+script: 
+  - bundle exec kitchen verify ${INSTANCE}
+
+before_deploy:
+  # Only deploy if the build passed successfully
+  - test $TRAVIS_TEST_RESULT = 0
```

Comment:
This section has already been removed from the `template-formula`, see https://github.com/saltstack-formulas/template-formula/pull/41/files#diff-354f30a63fb0907d4ad57269548329e3L20.  It has been replaced by defining the `stages` at the top of the linked file.